.github/scripts/cleanup_dbt_resources.sh

```diff
@@ -51,5 +51,12 @@ echo "$schemas"
 
 echo "$schemas" | xargs -i bash -c 'delete_database {}'
 
+s3_dependency_dir=$(dbt run-operation print_s3_dependency_dir --quiet \
+    --target "$1"
+)
+
+echo "Deleting Python model requirements at $s3_dependency_dir"
+aws s3 rm "$s3_dependency_dir" --recursive
```

Comment:
Hmmm alright, as long as there's no way for that call to ever target anything other than the target env buckets then I think it's okay. I can dig a little more on this when we do our permissions cleanup.

Reply:
Sounds good! I think we're also slightly more protected by the switch to variables for defining `s3_dependency_dir` in [02cc83a](https://github.com/ccao-data/data-architecture/pull/435/commits/02cc83a3ee89b3bda571f489d9f04a125ab069b3), since now our S3 paths for dependencies don't overlap with our paths for data.